src/java/htsjdk/samtools/CRAMFileWriter.java

```diff
@@ -46,8 +46,9 @@
 
 @SuppressWarnings("UnusedDeclaration")
 public class CRAMFileWriter extends SAMFileWriterImpl {
-    private static final int REF_SEQ_INDEX_NOT_INITIALIZED = -2;
+    private static final int REF_SEQ_INDEX_NOT_INITIALIZED = -3;
```

Comment:
@cmnbroad Ideally I'd like SAMRecord.MULTIPLE_REFERENCES_INDEX=-2 and then Slice and others could rely on SAMRecord's ref id space. Do you think this is possible?